xyz.aguno.CubeTimer.appdata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>xyz.aguno.CubeTimer.desktop</id>
```

Comment:
```suggestion
  <id>xyz.aguno.CubeTimer</id>
```

Which bit of documentation tells to add `.desktop` here. I'd like to get it fixed.